services/core/shared/core/delegates.js

```diff
@@ -124,7 +124,8 @@ const loadAllDelegates = async () => {
 	await BluebirdPromise.map(
 		delegateList,
 		async delegate => {
-			await cacheRedisDelegates.set(delegate.address, delegate);
+			const delegateAddress = delegate.address || delegate.account.address;
```

Comment:
Does it depend on lisk-sdk version? Maybe it is worth moving it to the respective compat directories?

Reply:
Yes, it is version dependent So, I moved that logic to `compat` layer. Thanks for the suggestion!